### PR TITLE
update packages and add functionality to delete pdf file

### PR DESCRIPTION
**Pull request by MorrisMalone**

### CODE REVIEW

File: src/views/Merge.vue [hunk 1]
```diff
@@ -1,94 +1,156 @@
...
+            // https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop
 
-	const mergePDFFile = await mergePDF.save();
-	download(mergePDFFile, "merged.pdf", "application/pdf");
-	this.isLoading = false;
-},
-mosaicPDF: async function(){
-	let xC = 2;
-	let yC = 5;
+            for await (const file of this.files)
```
Comment (Steve2955) [inline src/views/Merge.vue:89]: Interesting, I didn't even knew this language feature

File: src/views/Merge.vue [hunk 2]
```diff
@@ -1,94 +1,156 @@
...
+                            y: page.getHeight()*((yC-1)/yC) - (page.getHeight() / yC * y),
+                            width: page.getWidth()/xC,
+                            height: page.getHeight()/yC,
+                        });
+                    }
+                }
+            }
+            const mergePDFFile = await mosaicPDF.save();
+            download(mergePDFFile, "merged.pdf", "application/pdf");
+        },
```
Comment (Steve2955) [inline src/views/Merge.vue:125]: Something is broken here, but it's not your fault. It was broken since before you forked it.

**Review by Steve2955 — APPROVED** [review]
Thank you very much for your contribution. The new features are a great addition and I even learned a new thing.

File: src/views/Merge.vue (hunk 2 above)
Comment (MorrisMalone) [reply src/views/Merge.vue:125]: what is broken exactly? I don't get any error when I run it.

Comment (Steve2955) [reply src/views/Merge.vue:125]: The grid mode is broken at least for me, see #7 for the log message.

Comment (MorrisMalone) [reply src/views/Merge.vue:125]: It seems it is not working only with some pdfs. I got this error with only one pdf. All the others where working fine, even bigger ones.

Comment (Steve2955) [reply src/views/Merge.vue:125]: weird, didn't work for any pdfs I tried 🤔

**Pull request merged**